code/datums/mood_events/generic_negative_events.dm

```diff
@@ -380,11 +340,6 @@
 	description = "I feel off-balance without my tail."
 	mood_change = -2
 
-/datum/mood_event/tail_regained_right
-	description = "My tail is back, but that was traumatic..."
-	mood_change = -2
-	timeout = 5 MINUTES
```

Comment:
This is definitely supposed to be used so the fact that it's not probably means the event associated is bugged

Reply:
In other words I would rather fix the reason this is unused than delete it